app/views/registered_apps/show.html.erb

```diff
@@ -0,0 +1,3 @@
+<div class="cointainer">
+  <h2><%=@registered_app.name%>: <%=@registered_app.url%></h2>
+</div>
```

Comment:
I thought this was required of the user story, but maybe I misinterpreted. Under build view of this check point:

> Build out the read action and views as well. The read view should display the registered application's name and URL. Lastly, construct the delete action to de-register and destroy an existing application registration. 

Was this supposed to be in the users user#show, then?